tools/tdbg/commands.go

```diff
@@ -0,0 +1,743 @@
+// The MIT License
```

Comment:
The proper name of this file `workflow_commands,go`.

Reply:
there are more commands in the file than just workflow commands

Reply:
Then it should be splitted. You can do it in separate PR.